### ISSUE TITLE
can not setting model to eval when using EpochBasedRunner

### ISSUE THREAD

**Issue by smarengin** [open-mmlab/mmcv#964]
I need to freeze some params in my model, so I code like this:

>         for module in modules:
>             m = getattr(self, module)
>             m.eval()
>             for param in m.parameters():
>                 param.requires_grad = False

but after training some iters, I find the params are changed, especially for the running_mean in BatchNorm layer
when I check the code of EpochBasedRunner in mmcv, it just like below:

>     def train(self, data_loader, **kwargs):
>         self.model.train()
>         self.mode = 'train'

I think 'self.model.train()' change the status of the layers from 'eval' to 'train'. so I can not freeze the params(especially for running_mean and var in BN) when using mmcv
any body can help me? thanks

**Comment by zhouzaida** [comment]
hi, @smarengin , you can freeze it in your model like https://github.com/open-mmlab/mmclassification/blob/master/mmcls/models/backbones/mobilenet_v2.py#L267

**Comment by smarengin** [comment]
hi, @zhouzaida, it's sovled, thank you!
